### PR TITLE
circos: new package starting at 0.69-6

### PR DESCRIPTION
**Pull request by molecuul**
depends on perl packages introduced in #9760

### CODE REVIEW

File: var/spack/repos/builtin/packages/circos/package.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Circos(Package):
+    """Circos is a software package for visualizing data and information."""
+
+    homepage = "http://www.example.com"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/circos/package.py]: Needs a proper homepage.

File: var/spack/repos/builtin/packages/circos/package.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    depends_on('perl-params-validate', type='run')
+    depends_on('perl-readonly', type='run')
+    depends_on('perl-regexp-common', type='run')
+    depends_on('perl-set-intspan', type='run')
+    depends_on('perl-statistics-basic', type='run')
+    depends_on('perl-svg', type='run')
+    depends_on('perl-text-format', type='run')
+
+    def install(self, spec, prefix):
+        basedir = join_path(prefix.lib, 'circos')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/circos/package.py]: You can use `prefix.lib.circos` here.

File: var/spack/repos/builtin/packages/circos/package.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        install_tree('.', basedir)
+
+        mkdirp(prefix.bin)
+        symlink(join_path(basedir, 'bin/circos'),
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/circos/package.py]: You can use `basedir.bin.circos` here.

File: var/spack/repos/builtin/packages/circos/package.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    depends_on('perl-svg', type='run')
+    depends_on('perl-text-format', type='run')
+
+    def install(self, spec, prefix):
+        basedir = prefix.lib.circos
+        install_tree('.', basedir)
+
+        mkdirp(prefix.bin)
+        symlink(basedir.bin.circos,
+                join_path(prefix.bin, 'circos'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/circos/package.py]: You can also use `prefix.bin.circos` here.

Comment (molecuul) [reply var/spack/repos/builtin/packages/circos/package.py]: Oops, don't know how I missed that one.

**Pull request merged**